### DISCUSSION TITLE
Error: Unable to find Allure implementation when using allure decorators.

### DISCUSSION THREAD

**Discussion by shaked-bouktus** [allure-framework/allure2#2371] [Questions & Support]
Im trying to add allure decorator to my testing framework.
Using mocha as test framework, project written in TS.
Followed the installation steps in here:
https://www.npmjs.com/package/allure-decorators?activeTab=readme

tried to add decorator to a function, and im getting the error in the title when im running the test.

**Comment by baev** [comment]
Could you please share more information about the error? The sample project to reproduce the problem would be helpful as well.
